include/derivs.hxx

```diff
@@ -50,12 +90,9 @@
 ///                    If not given, defaults to DIFF_DEFAULT
 /// @param[in] region  What region is expected to be calculated
 ///                    If not given, defaults to RGN_NOBNDRY
-const Field3D DDX(const Field3D& f, CELL_LOC outloc = CELL_DEFAULT,
-                  const std::string& method = "DEFAULT", REGION region = RGN_NOBNDRY);
-inline const Field3D DDX(const Field3D& f, CELL_LOC outloc, DIFF_METHOD method,
-                         REGION region = RGN_NOBNDRY) {
-  return DDX(f, outloc, toString(method), region);
-};
+const Field3D DDX(const Field3D& f, CELL_LOC outloc = CELL_DEFAULT, const std::string&
+    method = "DEFAULT", const std::string& region = "RGN_NOBNDRY");
+DERIV_FUNC_REGION_ENUM_TO_STRING(DDX, Field3D)
```

Comment:
As we're touching this might, may as well drop the `const` from the return value (similarly in the macro)

Reply:
Do we want to drop the `const`s in `derivs.cxx` as well?

Reply:
Ignore that, compiler error tells me we do drop those `const`s too...

Reply:
I guess we want to do the same in `coordinates.hxx`/`coordinates.cxx` too? I've made that update in the last commit.